plugins/verifier/cosign/README.md

```diff
@@ -1,23 +1,30 @@
-## Cosign Verifier
+# Cosign Verifier
 
-This README outlines how this validation framework can be used to verify signatures generated using [cosign](https://github.com/sigstore/cosign/tree/cb0c46a429253287429868c3721c9f8693797114). The verifier is added as a plugin to the framework that uses [cosign](https://github.com/sigstore/cosign/tree/cb0c46a429253287429868c3721c9f8693797114) packages to invoke the verification of an image. Currently cosign verifier works with remote registry that can provide cosign related artifacts linked as specially formatted tag to the subject artifact. It works only with [ociregistry](../../referrerstore/ociregistry) referrer store plugin that uses the OCI registry API to discover and fetch the artifacts. 
+This README outlines how this validation framework can be used to verify signatures generated using [cosign](https://github.com/sigstore/cosign/). The verifier is added as a plugin to the framework that uses [cosign](https://github.com/sigstore/cosign/) packages to invoke the verification of an image. Currently cosign verifier works with remote registry that can provide cosign related artifacts linked as specially formatted tag to the subject artifact. It works only with [oras](../../pkg/referrerstore/oras) referrer store plugin that uses the OCI registry API to discover and fetch the artifacts.
 
-### Fallback in OCIRegistry store
-A configuration flag called ```cosign-enabled``` is introduced to the plugin configuration. If this flag is enabled, the ```ListReferrers``` API will attempt to query for the cosign signatures for a subject in addition to the references queried using ```referrers API```. All the cosign signatures are returned as the reference artifacts with the artifact type ```org.sigstore.cosign.v1``` This option will enable to verify cosign signatures against any registry including the onces that don't support the [notaryproject](https://github.com/notaryproject)'s ```referrers API```. 
+## Fallback in OCIRegistry store
+A configuration flag called `cosignEnabled` is introduced to the plugin configuration. If this flag is enabled, the `ListReferrers` API will attempt to query for the cosign signatures for a subject in addition to the references queried using `referrers API`. All the cosign signatures are returned as the reference artifacts with the artifact type `org.sigstore.cosign.v1` This option will enable to verify cosign signatures against any registry including the onces that don't support the [notaryproject](https://github.com/notaryproject)'s `referrers` API.
 
-IMPORTANT NOTE: Cosign signatures cannot be verified from private registries. ```cosign-enabled``` flag should not be enabled for any private registry scenario even for non-cosign signature verifications.
```

Comment:
cosign code path currently only works with registries with no Auth. Should we keep this note?

Reply:
added this back